test/test_with_xpress.jl

```diff
@@ -1440,3 +1440,138 @@ end
     end
 end
 
+# # equivalent REopt API Post for test 2:
```

Comment:
Not sure what that line refers to, I think it may be associated with the example POST in lines 1444-1495? I can remove those lines since they are commented out either ways.